services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java

```diff
@@ -885,9 +885,10 @@ private NewsFilter buildFilter(List<String> spaces, String filter, String text,
   }
   
   private boolean canCreateNews(String authenticatedUser, Space space) throws Exception {
+    org.exoplatform.services.security.Identity currentIdentity = ConversationState.getCurrent().getIdentity();
     return space != null && 
           (spaceService.isSuperManager(authenticatedUser) || spaceService.isManager(space, authenticatedUser) || spaceService.isRedactor(space, authenticatedUser) ||
-          spaceService.isMember(space, authenticatedUser) && ArrayUtils.isEmpty(space.getRedactors()));
+          spaceService.isMember(space, authenticatedUser) && ArrayUtils.isEmpty(space.getRedactors()) || currentIdentity.isMemberOf(PLATFORM_WEB_CONTRIBUTORS_GROUP, PUBLISHER_MEMBERSHIP_NAME));
```

Comment:
```suggestion
          spaceService.isMember(space, authenticatedUser) && (ArrayUtils.isEmpty(space.getRedactors() || currentIdentity.isMemberOf(PLATFORM_WEB_CONTRIBUTORS_GROUP, PUBLISHER_MEMBERSHIP_NAME)));
```
the publisher in web contributors group should be also a space member